web-components/src/components/floating-modal/FloatingModal.test.ts

```diff
@@ -170,6 +169,19 @@ describe("Floating Modal Component", () => {
 
     const mdButton = minimizedModal.shadowRoot!.querySelector(".md-floating__resize") as Button.ELEMENT;
     expect(mdButton.getAttribute("arialabel")).toEqual("Maximize Modal");
-  }
-  );
+  });
+  test("should center align when centered attribute is true", async () => {
```

Comment:
this PR has only test case changes?

Reply:
@merajcal implementation was merged here https://github.com/momentum-design/momentum-ui/pull/1949